gradle.properties

```diff
@@ -11,3 +11,6 @@ android.nonTransitiveRClass=true
 #MPP
 kotlin.mpp.enableCInteropCommonization=true
 kotlin.mpp.androidSourceSetLayoutVersion=2
+
+# Default flavor
```

Comment:
Following the other comment convention above, we don't need to add a space here 🙏 
`# Default flavor` -> `#Default flavor`